### PR TITLE
Import fenix quarantine

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Import fenix quarantine

### CODE REVIEW

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -2407,4 +2407,14 @@
     <string name="debug_drawer_tab_tools_tab_count_private">Private</string>
     <!-- The total tab count category in the tab count section in Tab Tools. -->
     <string name="debug_drawer_tab_tools_tab_count_total">Total</string>
+    <!-- The title of the tab creation tool section in Tab Tools. -->
```
Comment (flodolo) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml:2410]: Is it just me or this looks like a tool that is targeting developers, and shouldn't have localizable strings? Part of the strings also landed in the [previous batch](https://github.com/mozilla-l10n/android-l10n/pull/720/files#diff-b8d7151f11faa90ad8cfbb96437f96d05602954aa79c1145cb3232f6a5eb6d38R2397).

Comment (Delphine) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml:2412]: I think that's correct, I'll get an issue on file to double check

**Comment by Delphine** [comment]
@MozillaNoah: seems like you landed the Debug Drawer strings. Could you confirm these strings are for a tool that is targeting developers? If so, none of these should not be localizable, and you would need to remove them. Thanks!

**Comment by MozillaNoah** [comment]
> @MozillaNoah: seems like you landed the Debug Drawer strings. Could you confirm these strings are for a tool that is targeting developers? If so, none of these should not be localizable, and you would need to remove them. Thanks!

The tool will eventually be available inside Release, and we will likely have users use the tool when troubleshooting bugs with the Support team or on Bugzilla, so we opted to have those localized for that situation.

**Comment by Delphine** [comment]
> The tool will eventually be available inside Release, and we will likely have users use the tool when troubleshooting bugs with the Support team or on Bugzilla, so we opted to have those localized for that situation.

That makes sense, thanks for clarifying!

**Review by Delphine — APPROVED** [review]

**Pull request merged**